resources/keywords/import_provider_account.resource

```diff
@@ -39,12 +39,16 @@ User Navigates To Import Database Provider Account Screen From Database Access P
     ...        Will be removed once issue is fixed
 
 Select ${option} From ${menu} Button DropDown
-    ${btnMenuDropDownUpd}=    Replace String    ${btnMenuDropDown}    <<menu>>    ${menu}
-    Wait Until Keyword Succeeds    10s    1s    Wait Until Element Is Visible    ${btnMenuDropDownUpd}
-    Click Element    ${btnMenuDropDownUpd}
-    ${btnMenuOptDropDownUpd}=    Replace String    ${btnMenuOptDropDown}    <<menuopt>>    ${option}
-    Element Should Be Visible    ${btnMenuOptDropDownUpd}
-    Click Element    ${btnMenuOptDropDownUpd}
+    Wait Until Keyword Succeeds    10s    1s    Wait Until Element Is Visible    ${btnMenuDropDown}
+    Click Element    ${btnMenuDropDown}
+    Element Should Be Visible    ${btnMenuOptDropDown}
+    Click Element    ${btnMenuOptDropDown}
+
+Select ${option} From ${menu} Button DropDown Provision
```

Comment:
@ikanias As mentioned on the above, this is been already handled with existing keyword

Reply:
As mentioned in my previous reply, the 'menu' parameter is not working properly anymore and is not displayed in the DOM anymore

Reply:
Same as above